### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [Vanadiry/blog-comments#12] [blog-comments]
# about/

20年之后，更令你失望的不是你做了什么，而是你没做什么。所以解开帆脚索，离开安全的港湾，赶着航程中的信风，去探索，去梦想，去发现。新年倒计时更新时间线便笺  关于我Vanadiry 是普通人，好像什么都玩一点的样子。 住在 Rabbit House，心跳文学社的一员，有时候去海拉鲁旅游，也会去洛圣都抢点好玩的。有时候作为“岛民代表”去我的岛上捉鲈鱼和种花。 Vanadiry 不喜欢一些东西，但你不

https://magic.vanadiry.com/about/

**Comment by Vanadiry** [comment]
喵喵喵

**Comment by Vanadiry** [comment]
（悲）

**Comment by Vanadiry** [comment]
🎑中秋快乐！
